psydac/linalg/tests/test_iterative_solvers.py

```diff
@@ -0,0 +1,498 @@
+import numpy as np
+import pytest
+from psydac.linalg.iterative_solvers import bicg, bicgstab, lsmr, minres, pcg, jacobi, cg
+from psydac.linalg.stencil import StencilVectorSpace, StencilMatrix, StencilVector
+from psydac.linalg.basic import LinearSolver
+from psydac.ddm.cart import DomainDecomposition, CartDecomposition
```

Comment:
Please add two empty lines before the functions. See [here](https://peps.python.org/pep-0008/#blank-lines).

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)